tests/particle/test_particle.py

```diff
@@ -171,6 +171,10 @@ def test_from_nucleus_info():
     p = Particle.from_nucleus_info(1, 2, anti=True)
     assert p.pdgid == -1000010020
 
+    # proton and neutron should return the preferred quark representation
```

Comment:
```suggestion
def test_from_nucleus_info_special_cases():
    """
    The proton and the neutron should return the preferred quark representation
    rather than the representation as a nucleus.
```

Reply:
Indeed better to avoid too big test functions and have dedicated ones for special cases.